alan-turing-institute/rctab-infrastructure
# Update docs to reflect Azure TLS certificate changes

## Summary

The root CA that should be used (via the DB_ROOT_CERT_PATH setting) has changed. This PR updates docs to reflect that.

## How to Test

1. Try to deploy RCTab with the old file.
2. Observe that the rctab-api-user creation step fails.
3. Follow the new steps.
4. Re-run `pulumi up`, which should now pass.
